tests/Acceptance/exercicio8Cest.php

```diff
@@ -0,0 +1,53 @@
+<?php
+
+namespace Tests\Acceptance;
+
+use Tests\Support\AcceptanceTester;
+use Codeception\Example;
+
+class exercicio8Cest
+{
+    public function _before(AcceptanceTester $I)
+    {
+    }
+    public function TitlExercicio8Test(AcceptanceTester $I)
+    {
+        $I->amOnPage('/exercicio8');
+        $I->seeInTitle('Exercício 8');
+    }
+    public function testExercicio8(\Codeception\Actor $I, Example $data)
+    {
+        $I->amOnPage('/exercicio8');
+
+        $I->see('Digite um número:');
+        $I->fillField('#numero', $data['input']);
+        $I->click('Enviar');
+
+        if ($data['input'] >= 1) {
+            $I->see("Números de 1 até {$data['input']}:");
+            for ($i = 1; $i <= $data['input']; $i++) {
+                $I->see("$i ");
+            }
+
+            $produto = 1;
+            for ($i = 1; $i <= $data['input']; $i++) {
+                $produto *= $i;
+            }
+
+            $I->see("Produto dos números de 1 até {$data['input']}:");
+            $I->see($produto);
+        } else {
+            $I->see("O número deve ser maior ou igual a 1.");
+        }
+    }
+
+    protected function inputProvider()
+    {
+        return [
+            ['input' => 5], // Teste com número válido
+            ['input' => 0], // Teste com número inválido
+            ['input' => -3], // Teste com número inválido
```

Comment:
Isso que você fez ficou otimo!!! Apenas um detalhe para a cobertura de teste. Quando vc passa o valor dessa forma, o ideal é ja enviar o resultado que espera sem a necessidade do for na logica do teste.

Motivo disso nesse caso é que essa logica está no backend e foi copiado de la, se ela tiver errado dara um falso positivo.

No caso o ideal seria algo assim 
```
['input' => 5, 'result' => '1 2 3 4 5', 'product' => '120']
```